<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssaultCubeTrainerQTClass</class>
 <widget class="QWidget" name="AssaultCubeTrainerQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AssaultCube 1.3.0.2 Trainer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AssaultCubeTrainerQT.qrc">
    <normaloff>:/icons/25207.png</normaloff>:/icons/25207.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>120</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="healthCheckbox">
      <property name="text">
       <string>HEALTH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="armorCheckbox">
      <property name="text">
       <string>ARMOR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="grenadeCheckbox">
      <property name="text">
       <string>GRENADE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="rifleAmmoCheckbox">
      <property name="text">
       <string>RIFLE AMMO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="pistalAmmoCheckbox">
      <property name="text">
       <string>PISTAL AMMO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="sniperAmmoCheckbox">
      <property name="text">
       <string>SNIPER AMMO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="shotgunAmmoCheckbox">
      <property name="text">
       <string>SHOTGUN AMMO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>85</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="rapidFireCheckbox">
      <property name="text">
       <string>RAPID FIRE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="noRecoilCheckbox">
      <property name="text">
       <string>NO RECOIL</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>161</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="AssaultCubeTrainerQT.qrc">:/icons/25207.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>166</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Developer : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>un4ckn0wl3z</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>https://www.unknownclub.net</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="AssaultCubeTrainerQT.qrc">:/images/98973-nyan-cat-hd-image-free.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AssaultCubeTrainerQT.qrc"/>
 </resources>
 <connections/>
</ui>
